echo-webhooks/src/main/groovy/com/netflix/spinnaker/echo/scm/GithubWebhookEventHandler.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.echo.scm;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spinnaker.echo.model.Event;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+import static net.logstash.logback.argument.StructuredArguments.kv;
+
+@Component
+@Slf4j
+public class GithubWebhookEventHandler implements WebhookEventHandler{
+
+  private ObjectMapper objectMapper;
+
+  @Autowired
+  public GithubWebhookEventHandler(ObjectMapper objectMapper) {
+    this.objectMapper = objectMapper;
+  }
+
+  public boolean handles(String source) {
+    return source.equals("github");
```

Comment:
You should change this sort of check to `"github".equals(source)` as this inversion will always avoid an NPE in case `source` is null.